packages/app/src/app/pages/Sandbox/Editor/Workspace/items/More/More.tsx

```diff
@@ -36,7 +38,7 @@ export const More = observer(() => {
           <ProgressButton
             block
             loading={isForkingSandbox}
-            onClick={() => forkSandboxClicked()}
+            onClick={forkSandboxClicked}
```

Comment:
Did you test that this works without warnings? I'm not sure if Cerebral likes receiving an event.

Reply:
Not tested manually, but this is just basic JS, no? 🤔

Reply:
It is, I mean that the signal of Cerebral can receive an object. Cerebral will not execute the signal if that object is not serializable (like functions). I'm not sure if the argument of `onClick` is an event and whether it's serializable. That's probably why I used an arrow func here.

I could also be wrong though.

Reply:
Don't know enough of `Cerebral` to know what's happening, but this is the implementation of `forkSandboxClicked`:

```js
export const forceForkSandbox = [
  when(state`editor.currentSandbox.owned`),
  {
    true: [
      actions.confirmForkingOwnSandbox,
      {
        confirmed: forkSandbox,
        cancelled: [],
      },
    ],
    false: forkSandbox,
  },
];
```

Reply:
And  this is the used `confirmForkingOwnSandbox`:

```js
export function confirmForkingOwnSandbox({ browser, path }) {
  return browser.confirm('Do you want to fork your own sandbox?')
    ? path.confirmed()
    : path.cancelled();
}
```

Reply:
That makes sense, the signal executor passes the info. The best way to be sure is by just pressing the fork button at the "More" screen.

Reply:
I think the PR build is here: https://pr1987.build.csb.dev/s/vanilla. But it seems like there's an error: 
![image](https://user-images.githubusercontent.com/587016/58506770-7b92c200-8190-11e9-9e3e-a6545728a5c8.png)

(cc @lbogdan)